var/spack/repos/builtin/packages/serenity/package.py

```diff
@@ -24,7 +24,7 @@ class Serenity(CMakePackage):
     depends_on("cmake@3.12:", type="build")
     depends_on("boost")
     depends_on("eigen@3:")
-    depends_on("googletest@1.8.1:", type="test")
+    depends_on("googletest@1.8.1:", type="build")
     depends_on("hdf5@1.10.1:")
```

Comment:
While you're at it, could you add this as well?
```suggestion
    depends_on("hdf5@1.10.1: +hl +cxx")
```
See https://github.com/qcserenity/serenity/blob/607fa71da501b7c7e3f67cb19c064ba65a4f7feb/CMakeLists.txt#L110

Reply:
With this change I get the following failure
````
  >> 126    /tmp/awvwgk/spack-stage/spack-stage-serenity-1.4.0-dpss67esoc3i37ah5fuq35bodakiaujk/spack-src/src/data/ElectronicStructure.cpp:198:62: error: call of overloaded 'H5File(const char*, unsigned int, int)' is ambiguous
     127      198 |   HDF5::H5File file(name.c_str(), H5F_ACC_RDONLY, H5P_DEFAULT);
     128          |                                                              ^
     129    In file included from /home/awvwgk/projects/spack/gh/opt/spack/linux-ubuntu20.04-zen2/gcc-10.3.0/hdf5-1.14.0-ah57qce2zxrmcmdrhfuinoorkx3kv2zc/include/H5Cpp.h:47
````

Reply:
Me too... I'll try removing +HL

At least one of the two components was necessary for me though to get through the configure stage.

Weird also, since the cmakelists.txt seems to require both?

Reply:
Seems to be an issue with HDF5 1.14.0

Reply:
If I remove the `+hl` variant, I'm getting
```
    65      Could NOT find HDF5 (missing: HL) (found version "1.14.0")
```

I wonder how you managed to build serenity before?

Reply:
If you still have your build, could you provide the output of
```
spack find -v hdf5
```

Reply:
Cheers, somehow my Github page did not refresh and I missed your last comment